front/app/containers/IdeasShow/components/AIInsights/index.tsx

```diff
@@ -0,0 +1,125 @@
+// This code is a prototype for input authoring. Clean-up will follow after the prototype phase.
+import React, { useEffect, useState } from 'react';
+
+import {
+  Box,
+  Button,
+  Text,
+  Title,
+  colors,
+} from '@citizenlab/cl2-component-library';
+
+import useAddAuthoringAssistance from 'api/authoring_assistance/useAuthoringAssistance';
+
+import useFeatureFlag from 'hooks/useFeatureFlag';
+
+import IdeaCard from 'components/IdeaCard';
+import TextArea from 'components/UI/TextArea';
+
+import { useIntl } from 'utils/cl-intl';
+
+import messages from './messages';
+
+const AIInsights = ({ ideaId }: { ideaId: string }) => {
+  const { formatMessage } = useIntl();
+  const isAuthoringAssistanceOn = useFeatureFlag({
+    name: 'authoring_assistance',
+  });
+
+  const [customPrompt, setCustomPrompt] = useState<string>('');
+  const {
+    mutate: addAuthoringAssistance,
+    data: authoringResponse,
+    isLoading,
+  } = useAddAuthoringAssistance();
+  const promptData = authoringResponse?.data;
+  const similarIdeaIds =
```

Comment:
I don't fully understand how you can already access this data if the request didn't happen yet?